src/main/java/com/questhelper/quests/princealirescue/PrinceAliRescue.java

```diff
@@ -169,10 +169,10 @@ public void setupConditions()
 	{
 		inCell = new ZoneRequirement(cell);
 		hasWigPasteAndKey = new Conditions(dyedWig.alsoCheckBank(questBank), paste.alsoCheckBank(questBank), key.alsoCheckBank(questBank));
-		givenKeyMould = new Conditions(true, LogicType.OR,
+		givenKeyMould = new Conditions(true, LogicType.OR,	// TODO quest journal widget text outdated
```

Comment:
The first two WidgetTextRequirements will not refresh the quest progress when opening up the Quest Journal, because the text has changed. I was unable to find the current version of the text so these conditions will never trigger until it is fixed.